pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_scaler.go

```diff
@@ -96,6 +97,29 @@ func applyBounds(min, max, x int32) int32 {
 	return x
 }
 
+// GetScale returns the current scale resource for the PA.
+func (ks *kpaScaler) GetScale(ctx context.Context, pa *pav1alpha1.PodAutoscaler) (*autoscalingapi.Scale, error) {
+	logger := logging.FromContext(ctx)
+	resource, resourceName, err := scaleResourceArgs(pa)
+	if err != nil {
+		logger.Errorw("Unable to parse APIVersion", zap.Error(err))
```

Comment:
is logging needed? I feel like the caller should log.

Reply:
Degraded to Debug.